tests/v2/waku_relay/test_wakunode_relay.nim

```diff
@@ -208,7 +213,8 @@ procSuite "WakuNode - Relay":
 
     await allFutures(node1.stop(), node2.stop(), node3.stop())
 
-  asyncTest "Stats of peer sending wrong WakuMessages are updated":
+  # TODO: Add a function to validate the WakuMessage integrity
+  xasyncTest "Stats of peer sending wrong WakuMessages are updated":
```

Comment:
This test is disabled because the recently added validator has been commented out. Waiting to come up with a solution to re-enable it.